vowpalwabbit/cb_adf.cc

```diff
@@ -90,9 +93,11 @@ struct cb_adf
   void learn_DR(multi_learner& base, multi_ex& examples);
   void learn_DM(multi_learner& base, multi_ex& examples);
   void learn_SM(multi_learner& base, multi_ex& examples);
-
   template <bool predict>
   void learn_MTR(multi_learner& base, multi_ex& examples);
+
+  void save_prediction(const action_scores& a_s);
```

Comment:
Removed.  Predictions are saved in global_data.cc where learn() is called after predict().  Prediction is restored there after learn() is called since sometimes learn() changes predictions.